### PR TITLE
finish week5 but didnt make optional

### PR DESCRIPTION
**Pull request by DmitriiBogg**
I added the folders and code as required by the assignment, tested all the requests like GET and POST in Postman, and everything works. I didn't do the optional assignment, but I completed the LeetCode task.

### CODE REVIEW

File: 03-task-manager/starter/package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.11.10"
+    "mongodb": "^6.11.0",
```
Comment (chase-cove) [inline 03-task-manager/starter/package.json:15]: did you do anything with the mongodb driver?

File: 03-task-manager/starter/controllers/tasks.js
Comment (chase-cove) [inline 03-task-manager/starter/controllers/tasks.js:1]: this looks really great!

**Review by chase-cove — APPROVED** [review]
Hi Dmitrii,

This looks really great! It looks like you went above and beyond and did some of next week's assignment too! 

How did you find this lesson? Did you have any trouble with any of it?

Keep up the good work!

File: 03-task-manager/starter/package.json (hunk 1 above)
Comment (DmitriiBogg) [reply 03-task-manager/starter/package.json:15]: i made a update. may be that's why

**Pull request merged**